doc/source/cd/background.md

```diff
@@ -0,0 +1,862 @@
+
+- **Covariate drift**: Also referred to as input drift, this occurs
+  when the distribution of the input data has shifted
+  $P(\mathbf{X}) \ne P_{ref}(\mathbf{X})$, whilst
+  $P(\mathbf{Y}|\mathbf{X})$ =
+  $P_{ref}(\mathbf{Y}|\mathbf{X})$. The model may now be
+  extrapolating, potentially resulting in unreliable predictions.
+- **Prior drift**: Also referred to as label drift, this occurs when
+  the distribution of the outputs has shifted
+  $P(\mathbf{Y}) \ne P_{ref}(\mathbf{Y})$, whilst
+  $P(\mathbf{X}|\mathbf{Y})=P_{ref}(\mathbf{X}|\mathbf{Y})$. The
+  model could still be accurate, but accuracy metrics might be
+  affected.
+- **Concept drift**: This occurs when the process generating $y$
+  from $x$ has changed, such that
+  $P(\mathbf{Y}|\mathbf{X}) \ne P_{ref}(\mathbf{Y}|\mathbf{X})$.
+  It is possible that the model might no longer give a suitable
+  approximation of the true process.
+
+Note that a change in one of the conditional probabilities
+$P(\mathbf{X}|\mathbf{Y})$ and $P(\mathbf{Y}|\mathbf{X})$
+does not necessarily imply a change in the other. For example, consider
+the pneumonia prediction example from [Lipton et
+al.](https://arxiv.org/pdf/1802.03916.pdf), whereby a classification
+model $f$ is trained to predict $y$, the occurrence (or not)
+of pneumonia, given a list of symptoms $\boldsymbol{x}$. During a
+pneumonia outbreak, $P(\mathbf{Y}|\mathbf{X})$ (e.g. pneumonia
+given cough) might rise, but the manifestations of the disease
+$P(\mathbf{X}|\mathbf{Y})$ might not change.
+
+Below, the different types of drift are visualised for a simple
+two-dimensional classification problem. It is possible for a drift to
+fall under more than one category, for example the *prior drift* below
+also happens to be a case of *covariate drift*.
+
+```{image} images/bg_2d_drift.png
+:align: center
+:alt: 2D drift example
+```
+
+It is relatively easy to spot drift by eyeballing these figures here.
+However, the task becomes considerably harder for high-dimensional real
+problems, especially since real-time ground truths are not typically
+available. Some types of drift, such as prior and concept drift, are
+also difficult to detect without access to ground truths. As a
+workaround proxies are required, for example a model’s predictions can
+be monitored to check for prior drift.
+
+## 2. Detecting drift
+
+[Alibi Detect](https://github.com/SeldonIO/alibi-detect) offers a
+wide array of methods for detecting drift (see
+[here](../overview/algorithms.md)), some of which are examined in the
+NeurIPS 2019 paper [Failing Loudly: An Empirical Study of Methods for
+Detecting Dataset Shift](https://arxiv.org/abs/1810.11953).
+Generally, these aim to determine whether the distribution
+$P(\mathbf{x})$ has drifted from a reference distribution
+$P_{ref}(\mathbf{x})$, where $\mathbf{x}$ may represent
+input data $\mathbf{X}$, true output data $\mathbf{Y}$, or
+some form of model output, depending on what type of drift we wish to
+detect.
```

Comment:
I guess you prefer to keep using x rather than introduce a third variable such as Z but I think this could be a bit confusing give how lower case x usually denotes either constituents of a dataset X or realisations of a rv X.

Could be worth noting that this notation is unconventional, i.e. "...P_ref(x), where we abuse notation slightly to allow x to represent..."

Reply:
Also had exactly the same thought as this. I've changed to $\mathbf{z}$ now, and updated figures etc accordingly. Reads OK I think but I need to check again for consistency.